account/manager.go

```diff
@@ -107,6 +108,65 @@ func (wt witnessType) witnessSize() (int, error) {
 	}
 }
 
+// Action is a type of account modification.
+type Action string
+
+const (
+	CREATE   Action = "create"
+	DEPOSIT  Action = "deposit"
+	WITHDRAW Action = "withdraw"
+	RENEW    Action = "renew"
+	CLOSE    Action = "close"
+)
+
+type TxLabel struct {
+	Account AccountTxLabel `json:"account"`
+}
+
+type AccountTxLabel struct {
+	Key           string         `json:"key"`
+	Action        Action         `json:"action"`
+	ExpiryHeight  uint32         `json:"expiry_height"`
+	OutputIndex   uint32         `json:"output_index"`
+	IsExpirySpend bool           `json:"expiry_spend"`
+	TxFee         btcutil.Amount `json:"tx_fee"`
+	BalanceDiff   btcutil.Amount `json:"balance_diff"`
+}
+
+// actionTxLabel returns a transaction label for use with account
+// modification actions.
+func actionTxLabel(account Account, action Action, isExpirySpend bool,
```

Comment:
Can just pass in a pointer to the account, no need to de-reference.

Reply:
My thought process: `account Account` would make it clear that `actionTxLabel` does not modify `account`.

But I can see that passing pointers is common in the codebase. Will change as you suggest.